### DISCUSSION TITLE
grafica valutazione

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#661] [Idee e proposte]
proposta grafica valutazioni con media e data subito evidente 
<img width="743" alt="Screenshot 2024-03-05 alle 22 05 41" src="https://github.com/iisgiua/giuaschool/assets/93823125/64f09532-01bd-42d4-b4b7-6d772c564838">

**Comment by trinko** [comment]
#
La segno come richiesta.
